Public/Src/Cache/ContentStore/Interfaces/FileSystem/AbsolutePath.cs

```diff
@@ -36,6 +36,26 @@ public class AbsolutePath : PathBase, IEquatable<AbsolutePath>
         /// </summary>
         public static int MinimumPathLength = OperatingSystemHelper.IsWindowsOS ? 2 : 1;
 
+        /// <summary>
+        /// Create an AbsolutePath for the root of a drive.
+        /// </summary>
+        public static AbsolutePath RootPath(char driveLetter = default(char))
+        {
+            if (OperatingSystemHelper.IsWindowsOS)
+            {
+                if (driveLetter == default(char))
+                {
+                    throw new ArgumentException(nameof(driveLetter));

```

Comment:
Should it be more descriptive error message in this case?

And I'm not sure about using default arg here... Maybe two overloads are better when one of them is not supported by one platform and another is not supported by another.

For instance, right now for unix the argument is just ignored, but I would say it should be similar exception.